src/registry_schemas/schemas/filing.json

```diff
@@ -194,6 +194,18 @@
           }
         }
       }
+    },
+    "alteration_property": {
+      "$id": "#alteration_property",
+        "type": "object",
+        "required": [
+          "alteration"
+        ],
+        "properties": {
+          "alteration": {
+            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/alteration"
+          }
+        }
```

Comment:
The `alteration` property could have been embedded into the alteration schema directly and it does seem like a lot of the previous schemas like `annual_report` do this.  The `alteration` and `incorporationApplication` schema don't do this.  I think it makes more sense to do it the `alteration` and `incorporationApplication` way as the `filing` schema or whatever referencing schema can decide how it uses the schema whether it be as a separate property or on the same level of the referencing schema.  

There is a lack of consistency in the schema definitions so we may want to update everything to follow one approach at some point.